erpnext/manufacturing/doctype/work_order/work_order_calendar.js

```diff
@@ -2,13 +2,11 @@
 // For license information, please see license.txt
 
 frappe.views.calendar["Work Order"] = {
-	fields:  ["planned_start_date", "planned_end_date", "status", "produced_qty", "qty", "name", "name"],
```

Comment:
What does this thing do anyway? Does it work like the `_list.js` setup?

Reply:
this is the configuration for calendar view